src/main/java/org/folio/circulation/support/StreamToListMapper.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.circulation.support;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class StreamToListMapper {
+  private StreamToListMapper() { }
+
+  public static <T> List<T> toList(Stream<T> stream) {
+    if (stream == null) {
+      return null;
```

Comment:
This Mapper class is intended for re-use and should have 100 % test coverage.

Reply:
Fully agree, unit tests will be added.